deploy/aliyun/manifests/db-monitor.yaml.example

```diff
@@ -0,0 +1,86 @@
+apiVersion: pingcap.com/v1alpha1
```

Comment:
what are example files for?

Reply:
These the CRs that including the special configuration on ACK, e.g. LB config and storage class, I will refer to these examples in the doc.